tests/utils/test_db.py

```diff
@@ -0,0 +1,201 @@
+from datetime import date
+from unittest.mock import patch
+
+from freezegun import freeze_time
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+    get_urls,
+)
+
+
+@freeze_time("2024-05-01")
+def test_create_url(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    # when
+    url_id = create_url(url_name)
+
+    # then
+    created_url = get_url_data(url_id)
+    assert created_url == [url_id, url_name, date.today()]
+
+
+@freeze_time("2024-05-01")
+def test_get_url_data(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    url_id = create_url(url_name)
+
+    # when
+    url_data_result = get_url_data(url_id)
+
+    # then
+    assert url_data_result == [url_id, url_name, date.today()]
+
+
+def test_get_url_data_none(test_database: None):
+    # given
+    url_id = -1
+
+    # when
+    url_data_result = get_url_data(url_id)
+
+    # then
+    assert url_data_result is None
+
+
+def test_get_url_by_name(test_database: None):
+    # given
+    url_name = "https://example.com"
+    url_id = create_url(url_name)
+
+    # when
+    found_url_id = get_url_by_name(url_name)
+
+    # then
+    assert found_url_id == url_id
+
+
+def test_get_url_by_name_none(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    # when
+    url_id = get_url_by_name(url_name)
+
+    # then
+    assert url_id is None
+
+
+def test_get_urls(test_database: None):
+    # given
+    url_name1 = "https://example1.com"
+    url_name2 = "https://example2.com"
+
+    create_url(url_name1)
+    create_url(url_name2)
+
+    # when
+    urls_data = get_urls()
+
+    # then
+    for url_data in urls_data:
+        assert url_data["url_name"] in (url_name1, url_name2)
+
+
+def test_get_urls_empty(test_database: None):
+    # given (no urls in DB)
+
+    # when
+    urls_data = get_urls()
+
+    # then
+    assert len(urls_data) == 0
```

Comment:
wdyt, will it be better to compare result to the empty list? `len(...)` doesn't say anything about the returned data as it works for strings, tuples, and other data types